[Bug] [core] flink run 原生方式提交时报错 ChunJunSqlParseException

### Search before asking

- [X] I had searched in the [issues](https://github.com/DTStack/chunjun/issues) and found no similar issues.


### What happened

当 以 flink run 原生方式提交时报错 ChunJunSqlParseException ，无法解析job内容。

```

------------------------------------------------------------
 The program finished with the following exception:

org.apache.flink.client.program.ProgramInvocationException: The main method caused an error: com.dtstack.chunjun.throwable.ChunJunSqlParseException: 
----------sql start---------
1>    /data01/chunjun-master/chunjun-examples/sql/stream/stream.sql

----------sql end--------- 

SQL parse failed. Encountered "/" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CREATE" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "DROP" ...
    "ELEMENT" ...
    "EVERY" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERSECTION" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SOME" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <HYPHENATED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    "BEGIN" ...
    "END" ...
    "SHOW" ...
    "USE" ...
    "EXECUTE" ...
    
        at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:372)
        at org.apache.flink.client.program.PackagedProgram.invokeInteractiveModeForExecution(PackagedProgram.java:222)
        at org.apache.flink.client.ClientUtils.executeProgram(ClientUtils.java:98)
        at org.apache.flink.client.cli.CliFrontend.executeProgram(CliFrontend.java:843)
        at org.apache.flink.client.cli.CliFrontend.run(CliFrontend.java:240)
        at org.apache.flink.client.cli.CliFrontend.parseAndRun(CliFrontend.java:1087)
        at org.apache.flink.client.cli.CliFrontend.lambda$main$10(CliFrontend.java:1165)
        at java.security.AccessController.doPrivileged(Native Method)
        at javax.security.auth.Subject.doAs(Subject.java:422)
        at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1657)
        at org.apache.flink.runtime.security.contexts.HadoopSecurityContext.runSecured(HadoopSecurityContext.java:41)
        at org.apache.flink.client.cli.CliFrontend.main(CliFrontend.java:1165)
Caused by: com.dtstack.chunjun.throwable.ChunJunRuntimeException: com.dtstack.chunjun.throwable.ChunJunSqlParseException: 
----------sql start---------
1>    /data01/chunjun-master/chunjun-examples/sql/stream/stream.sql

----------sql end--------- 

SQL parse failed. Encountered "/" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CREATE" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "DROP" ...
    "ELEMENT" ...
    "EVERY" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERSECTION" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SOME" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <HYPHENATED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    "BEGIN" ...
    "END" ...
    "SHOW" ...
    "USE" ...
    "EXECUTE" ...
    
        at com.dtstack.chunjun.Main.exeSqlJob(Main.java:139)
        at com.dtstack.chunjun.Main.main(Main.java:105)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:355)
        ... 11 more
Caused by: com.dtstack.chunjun.throwable.ChunJunSqlParseException: 
----------sql start---------
1>    /data01/chunjun-master/chunjun-examples/sql/stream/stream.sql

----------sql end--------- 

SQL parse failed. Encountered "/" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CREATE" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "DROP" ...
    "ELEMENT" ...
    "EVERY" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERSECTION" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SOME" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <HYPHENATED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    "BEGIN" ...
    "END" ...
    "SHOW" ...
    "USE" ...
    "EXECUTE" ...
    
        at com.dtstack.chunjun.sql.parser.SqlParser.lambda$parseSql$1(SqlParser.java:67)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
        at com.dtstack.chunjun.sql.parser.SqlParser.parseSql(SqlParser.java:61)
        at com.dtstack.chunjun.Main.exeSqlJob(Main.java:130)
        ... 17 more
Caused by: org.apache.flink.table.api.SqlParserException: SQL parse failed. Encountered "/" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CREATE" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "DROP" ...
    "ELEMENT" ...
    "EVERY" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERSECTION" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SOME" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <HYPHENATED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    "BEGIN" ...
    "END" ...
    "SHOW" ...
    "USE" ...
    "EXECUTE" ...
    
        at org.apache.flink.table.planner.parse.CalciteParser.parseSqlList(CalciteParser.java:82)
        at org.apache.flink.table.planner.delegation.ParserImpl.parse(ParserImpl.java:102)
        at org.apache.flink.table.api.internal.TableEnvironmentImpl.executeSql(TableEnvironmentImpl.java:723)
        at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:48)
        at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:45)
        at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:45)
        at com.dtstack.chunjun.sql.parser.AbstractStmtParser.handleStmt(AbstractStmtParser.java:45)
        at com.dtstack.chunjun.sql.parser.SqlParser.lambda$parseSql$1(SqlParser.java:64)
        ... 28 more
Caused by: org.apache.calcite.sql.parser.SqlParseException: Encountered "/" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CREATE" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "DROP" ...
    "ELEMENT" ...
    "EVERY" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERSECTION" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SOME" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <HYPHENATED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    "BEGIN" ...
    "END" ...
    "SHOW" ...
    "USE" ...
    "EXECUTE" ...
    
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.convertException(FlinkSqlParserImpl.java:483)
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.normalizeException(FlinkSqlParserImpl.java:246)
        at org.apache.calcite.sql.parser.SqlParser.handleException(SqlParser.java:140)
        at org.apache.calcite.sql.parser.SqlParser.parseStmtList(SqlParser.java:195)
        at org.apache.flink.table.planner.parse.CalciteParser.parseSqlList(CalciteParser.java:77)
        ... 35 more
Caused by: org.apache.flink.sql.parser.impl.ParseException: Encountered "/" at line 1, column 1.
Was expecting one of:
    "ABS" ...
    "ALTER" ...
    "ARRAY" ...
    "AVG" ...
    "CALL" ...
    "CARDINALITY" ...
    "CASE" ...
    "CAST" ...
    "CEIL" ...
    "CEILING" ...
    "CHAR_LENGTH" ...
    "CHARACTER_LENGTH" ...
    "CLASSIFIER" ...
    "COALESCE" ...
    "COLLECT" ...
    "CONVERT" ...
    "COUNT" ...
    "COVAR_POP" ...
    "COVAR_SAMP" ...
    "CREATE" ...
    "CUME_DIST" ...
    "CURRENT" ...
    "CURRENT_CATALOG" ...
    "CURRENT_DATE" ...
    "CURRENT_DEFAULT_TRANSFORM_GROUP" ...
    "CURRENT_PATH" ...
    "CURRENT_ROLE" ...
    "CURRENT_SCHEMA" ...
    "CURRENT_TIME" ...
    "CURRENT_TIMESTAMP" ...
    "CURRENT_USER" ...
    "CURSOR" ...
    "DATE" ...
    "DELETE" ...
    "DENSE_RANK" ...
    "DESCRIBE" ...
    "DROP" ...
    "ELEMENT" ...
    "EVERY" ...
    "EXISTS" ...
    "EXP" ...
    "EXPLAIN" ...
    "EXTRACT" ...
    "FALSE" ...
    "FIRST_VALUE" ...
    "FLOOR" ...
    "FUSION" ...
    "GROUPING" ...
    "HOUR" ...
    "INSERT" ...
    "INTERSECTION" ...
    "INTERVAL" ...
    "JSON_ARRAY" ...
    "JSON_ARRAYAGG" ...
    "JSON_EXISTS" ...
    "JSON_OBJECT" ...
    "JSON_OBJECTAGG" ...
    "JSON_QUERY" ...
    "JSON_VALUE" ...
    "LAG" ...
    "LAST_VALUE" ...
    "LEAD" ...
    "LEFT" ...
    "LN" ...
    "LOCALTIME" ...
    "LOCALTIMESTAMP" ...
    "LOWER" ...
    "MATCH_NUMBER" ...
    "MAX" ...
    "MERGE" ...
    "MIN" ...
    "MINUTE" ...
    "MOD" ...
    "MONTH" ...
    "MULTISET" ...
    "NEW" ...
    "NEXT" ...
    "NOT" ...
    "NTH_VALUE" ...
    "NTILE" ...
    "NULL" ...
    "NULLIF" ...
    "OCTET_LENGTH" ...
    "OVERLAY" ...
    "PERCENT_RANK" ...
    "PERIOD" ...
    "POSITION" ...
    "POWER" ...
    "PREV" ...
    "RANK" ...
    "REGR_COUNT" ...
    "REGR_SXX" ...
    "REGR_SYY" ...
    "RESET" ...
    "RIGHT" ...
    "ROW" ...
    "ROW_NUMBER" ...
    "RUNNING" ...
    "SECOND" ...
    "SELECT" ...
    "SESSION_USER" ...
    "SET" ...
    "SOME" ...
    "SPECIFIC" ...
    "SQRT" ...
    "STDDEV_POP" ...
    "STDDEV_SAMP" ...
    "SUBSTRING" ...
    "SUM" ...
    "SYSTEM_USER" ...
    "TABLE" ...
    "TIME" ...
    "TIMESTAMP" ...
    "TRANSLATE" ...
    "TRIM" ...
    "TRUE" ...
    "TRUNCATE" ...
    "UNKNOWN" ...
    "UPDATE" ...
    "UPPER" ...
    "UPSERT" ...
    "USER" ...
    "VALUES" ...
    "VAR_POP" ...
    "VAR_SAMP" ...
    "WITH" ...
    "YEAR" ...
    <UNSIGNED_INTEGER_LITERAL> ...
    <APPROX_NUMERIC_LITERAL> ...
    <DECIMAL_NUMERIC_LITERAL> ...
    <BINARY_STRING_LITERAL> ...
    <QUOTED_STRING> ...
    <PREFIXED_STRING_LITERAL> ...
    <UNICODE_STRING_LITERAL> ...
    <BIG_QUERY_DOUBLE_QUOTED_STRING> ...
    <BIG_QUERY_QUOTED_STRING> ...
    "(" ...
    <LBRACE_D> ...
    <LBRACE_T> ...
    <LBRACE_TS> ...
    <LBRACE_FN> ...
    "?" ...
    "+" ...
    "-" ...
    <BRACKET_QUOTED_IDENTIFIER> ...
    <QUOTED_IDENTIFIER> ...
    <BACK_QUOTED_IDENTIFIER> ...
    <HYPHENATED_IDENTIFIER> ...
    <IDENTIFIER> ...
    <UNICODE_QUOTED_IDENTIFIER> ...
    "BEGIN" ...
    "END" ...
    "SHOW" ...
    "USE" ...
    "EXECUTE" ...
    
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.generateParseException(FlinkSqlParserImpl.java:44417)
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.jj_consume_token(FlinkSqlParserImpl.java:44228)
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.SqlStmt(FlinkSqlParserImpl.java:4037)
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.SqlStmtList(FlinkSqlParserImpl.java:2934)
        at org.apache.flink.sql.parser.impl.FlinkSqlParserImpl.parseSqlStmtList(FlinkSqlParserImpl.java:298)
        at org.apache.calcite.sql.parser.SqlParser.parseStmtList(SqlParser.java:193)
        ... 36 more

```

### What you expected to happen

希望可以用 flink run 原生方式正常提交任务

### How to reproduce


提交命令如下：

```

/data01/chunjun-master/flink-1.16.1/bin/flink run -m yarn-cluster -yjm 8192m -ytm 3072m -ynm flink_ri_job -yqu hdp-test-test -p 1 -j /data01/chunjun-master/flink-1.16.1/lib/chunjun-core.jar -c com.dtstack.chunjun.Main -flinkLibDir /data01/chunjun-master/flink-1.16.1/lib -jobType sql -flinkConfDir /data01/chunjun-master/flink-1.16.1/conf -pluginLoadMode shipfile -mode yarn-per-job -job /data01/chunjun-master/chunjun-examples/sql/stream/stream.sql

```

### Anything else

_No response_

### Version

master

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/chunjun-core/src/main/java/com/dtstack/chunjun/Main.java
+++ b/chunjun-core/src/main/java/com/dtstack/chunjun/Main.java
@@ -72,8 +72,11 @@ import org.apache.flink.table.types.DataType;
 
 import com.google.common.base.Preconditions;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.StringUtils;
 
+import java.io.File;
+import java.io.IOException;
 import java.net.URL;
 import java.net.URLDecoder;
 import java.nio.charset.StandardCharsets;
@@ -91,8 +94,19 @@ public class Main {
         log.info("-------------------------------------------");
 
         Options options = new OptionParser(args).getOptions();
-        String job = URLDecoder.decode(options.getJob(), StandardCharsets.UTF_8.name());
-        String replacedJob = JobUtil.replaceJobParameter(options.getP(), job);
+        String replacedJob = "";
+        File file = new File(options.getJob());
+        if (file.isFile()) {
+            try {
+                replacedJob = FileUtils.readFileToString(file, StandardCharsets.UTF_8.name());
+            } catch (IOException ioe) {
+                log.error("Can not get the job info !!!", ioe);
+                throw new RuntimeException(ioe);
+            }
+        } else {
+            String job = URLDecoder.decode(options.getJob(), StandardCharsets.UTF_8.name());
+            replacedJob = JobUtil.replaceJobParameter(options.getP(), job);
+        }
         Properties confProperties = PropertiesUtil.parseConf(options.getConfProp());
         StreamExecutionEnvironment env = EnvFactory.createStreamExecutionEnvironment(options);
         StreamTableEnvironment tEnv =

